packages/react-core/src/components/ContextSelector/ContextSelectorToggle.tsx

```diff
@@ -57,7 +57,7 @@ export class ContextSelectorToggle extends React.Component<ContextSelectorToggle
 
   onDocClick = (event: any) => {
     const { isOpen, parentRef, onToggle } = this.props;
-    if (isOpen && parentRef && !parentRef.contains(event.target)) {
+    if (isOpen && parentRef.current && !parentRef.current.contains(event.target)) {
```

Comment:
I think you need to check that parentRef is defined here. that's what's causing the test to fail